cmd/mysterium_client/run/command.go

```diff
@@ -98,3 +105,13 @@ func (cmd *CommandRun) Kill() error {
 
 	return nil
 }
+
+func sigtermListener(terminator chan os.Signal, cmd *CommandRun) {
+	<-terminator
+	err := cmd.Kill()
+	if err != nil {
+		fmt.Printf("Unable to kill one of subroutines %q\n", err.Error())
+	}
+	fmt.Println("Good bye")
+	os.Exit(1)
```

Comment:
`Exit(1)` mean program ending with errorNo=1, success graceful stop  should end with errorNo=0

Reply:
agree - we are already doing `os.Exit(0)` in `mysterium_client.go`